components/nds-filters/src/FilterSummary.js

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import PropTypes, { elementType } from "prop-types";
+import classnames from "classnames";
+import { Tag } from "@nice-digital/nds-tag";
+import RemoveIcon from "@nice-digital/icons/lib/Remove";
+import "./../scss/filter-summary.scss";
+import validate from "validate-npm-package-name";
```

Comment:
What is this doing here? It seemed to cause issues when I installed the latest alpha version?